src/schema/v2/me/myCollectionUpdateArtworkMutation.ts

```diff
@@ -79,7 +79,7 @@ export const myCollectionUpdateArtworkMutation = mutationWithClientMutationId<
 
     try {
       const response = await myCollectionUpdateArtworkLoader(artworkId, {
-        artist_ids: artistIds,
+        artists: artistIds,
```

Comment:
@jonallured - sorry for removing MOG, just want to discuss a bit. Can you explain this change a bit?

Reply:
Discussion here: https://artsy.slack.com/archives/CS3H1K4BD/p1601490192118100